playbooks/roles/stunnel/meta/main.yml

```diff
@@ -1,3 +1,4 @@
 ---
 dependencies:
+  - { role: ufw }
```

Comment:
Should there also be a dependency on the `openvpn` role? Maybe it doesn't matter since it's only being used from an `include_role` in the `openvpn` role? Thoughts?